### PR TITLE
Feature/uc tables

### PR DESCRIPTION
**Pull request by YetAnotherMatt**

### CODE REVIEW

**Review by gethyn1 — APPROVED** [review]

**Review by dwybourn — APPROVED** [review]

File: db/src/main/resources/db/migration/V1_002__UC_datafeed_initial_format.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
+CREATE TABLE dwp_uc_household (
+    id uuid not null primary key,
+    household_identifier varchar(50),
+    file_import_number integer,
+    award_date date,
+    last_assessment_period_start date,
+    last_assessment_period_end date,
+    pregnancy_flag varchar(1),
+    earnings_threshold_exceeded_flag varchar(1),
+    no_of_children_under_four integer,
```
Comment (mikeSmith08) [inline db/src/main/resources/db/migration/V1_002__UC_datafeed_initial_format.sql:10]: Do we also need number of children under 1, or is that just used/collected by us on the application?

File: db/src/main/resources/db/migration/V1_003__legacy_datafeed_initial_format.sql [hunk 2]
```diff
@@ -0,0 +1,32 @@
+CREATE TABLE dwp_legacy_household (
+    id uuid not null primary key,
+    household_identifier varchar(50),
+    file_import_number integer,
+    award varchar(500),
```
Comment (mikeSmith08) [inline db/src/main/resources/db/migration/V1_003__legacy_datafeed_initial_format.sql:5]: What is the `award` column for?

**Comment by mikeSmith08** [comment]
This may just be me not having as much understanding of the area, but why do we need a separate set of tables for the legacy feed, the data is similar for both feeds, could we not have them in the same table with a legacy flag or similar? Probably not a good idea but I'm interested in your opinion and reasoning anyway.

File: db/src/main/resources/db/migration/V1_002__UC_datafeed_initial_format.sql (hunk 1 above)
Comment (YetAnotherMatt) [reply db/src/main/resources/db/migration/V1_002__UC_datafeed_initial_format.sql:10]: no_of_children_under_four is provided by the feed - that's the only reason it's there. We can use the dwp_uc_child table to count children.

File: db/src/main/resources/db/migration/V1_003__legacy_datafeed_initial_format.sql (hunk 2 above)
Comment (YetAnotherMatt) [reply db/src/main/resources/db/migration/V1_003__legacy_datafeed_initial_format.sql:5]: I believe this is the type of benefit they're recieving, but as I wasn't completely sure I left it with the wording from the document

**Review by mikeSmith08 — APPROVED** [review]

**Pull request merged**